src/licensedcode/data/licenses/Scilab-old.yml

```
@@ -0,0 +1,8 @@
+key: Scilab-old
+short_name: SCILAB
+name: SCILAB License 
+category: Proprietary Free
+owner: INRIA/ENPC
+homepage_url:  https://fedoraproject.org/wiki/Licensing/Scilab License Old
```

The URL is incorrect with spaces. It should be :
```suggestion
homepage_url:  https://fedoraproject.org/wiki/Licensing/Scilab_License_Old
```

Bu this is not the homepage_url:  but rather some place other URL.
The homepage was at http://web.archive.org/web/20051212214843/http://www.scilab.org/legal/license.html instead

---

Thanks for the homepage_url . I would make the change

---

:+1: